src/components/group-common.js

```diff
@@ -87,7 +87,7 @@ export const MemberCard = memoWithName('MemberCard', ({ member: { email, roles }
   }, [
     div({ role: 'rowheader', style: { flex: '1', whiteSpace: 'nowrap', textOverflow: 'ellipsis', overflow: 'hidden', height: '1rem' } }, [email]),
     div({ role: 'cell', style: { flex: '1', textTransform: 'capitalize', height: '1rem' } }, [_.includes(adminLabel, roles) ? adminLabel : userLabel]),
-    div({ role: 'cell', style: { flex: 'none' } }, [
+    isOwner && div({ role: 'cell', style: { flex: 'none' } }, [
```

Comment:
~This menu is now only shown if `isOwner` is true, but was previously enabled when `canEdit` is true. Is there a need for `canEdit` now? Will this menu ever be shown but disabled?~

Looks like this happens when the current user is the only owner of a billing project.

Reply:
Also, since this div is a table cell (`role: 'cell'`), it may be better to conditionally render the menu instead of the entire cell. That way, the number of columns would be consistent between table rows and match the number of columns in the table header. Though I'm not sure if it matters since this is the last column in the row.